doc/cygnus-ngsi/installation_and_administration_guide/reliability.md

```diff
@@ -0,0 +1,50 @@
+
+Dispatching policy regarding new incoming NGSI events and candidate for retries is currently hardcoded. It is based on always giving the priority to retries (if any), and then process new incoming events (if any). This could change in the future in the form of a configurable policy. In any case, the `batch_retry_intervals` parameter grants new incoming events are always processed in between candidates for retry, even if `batch_ttl` is set to `-1` (always retry).
+
+[Top](#top)
+
+## <a name="section2"></a>Traffic absorption peaks
+Because Cygnus inherits from Apache Flume, Cygnus agents of any type inherit an internal architecture based on the usage of channels communicating sources and sinks. Apart for achieving this communication, channels, which can be seen as buffers for incoming data, work as a mechanism for traffic absorption peaks.
+
+In the specific case of Cygnus for NGSI, this means the channel will be able to absorb peaks of notifications from a NGSI source (typically Orion Context Broker) while the configured sinks work hard in the persistence of data. Of course, timing regarding reception time and persistence time will be different, since sinks will work slower than sources (typical consuming-producer problem), but al least data will not be lost.
+
+A Cygnus administrator must carefully study the expected throughput for the whole architecture containing Cygnus and prevent high loads of NGSI notifications by configuring a channel capacity enough to absorb it.
+
+[Top](#top)
+
+## <a name="section3"></a>File channels
+File channels are another heritage from Apache Flume. This kind of channels are based on files, and unlike memory channels, information within this kind of channel lives between crashes.
+
+Using file channels is interesting if your notification rate is slower than Cygnus recovery time. Then, nothing will be lost. If not, using file channels or any other *volatile* channel will not imply any significant difference, since the amount of new incoming events lost while Cygnus is down could be much higher than the number of past events saved in files.
+
+File channels are particularly relevant if your aim is to implement some kind of High Availability architecture (see next section).
+
+As the reader may imagine, file-based channels are slower than memory-based ones, since file seeking is slow. Thus, you must find a trade-off between reliability and speed when deciding the most suitable channel type for your agent.
+
+[Top](#top)
+
+## <a name="section4"></a>High Availability
```

Comment:
Ummm, let me think about this, since this architecture doc could be outdated. A quick check shows me there are other documents better explaining advanced architectures, for instance. Most problably, I'll include in this architecture doc links to those other documents, including the reliability one.